### DISCUSSION TITLE
Do not install on Mac OS Ventura 13.3.1 (a) (22E772610a)

### DISCUSSION THREAD

**Discussion by lexkosha** [Eugeny/tabby#8834] [Release Notes]
Hi guys ;)
When installing on Mac OS Ventura 13.3.1 (a) (22E772610a), I get the answer "You cannot install this application on this computer. Contact the application administrator"

**Comment by liangweishen** [comment]
It is possible that the wrong installation package is selected. Inter chip and M1 chip are different architectures.
